dev/ci/user-overlays/09173-ejgallego-proofview+proof_info.sh

```diff
@@ -0,0 +1,12 @@
+if [ "$CI_PULL_REQUEST" = "9173" ] || [ "$CI_BRANCH" = "proofview+proof_info" ]; then
+
+    ltac2_CI_REF=proofview+proof_info
+    ltac2_CI_GITURL=https://github.com/ejgallego/ltac2
+
+    equations_CI_REF=proofview+proof_info
```

Comment:
@ejgallego can you remove the overlay if it is not needed anymore? This way, we'll get a final ok from CI.

Please also remove the unrelated Dune change.

Reply:
OK, rebasing.

Reply:
Done thanks!